src/Main.elm

```diff
@@ -67,20 +88,43 @@ init =
 -- Update
 
 
+toUpper : String -> String
+toUpper str =
+    String.toUpper (String.left 1 str) ++ String.dropLeft 1 str
+
+
+matchSelectedGenres : Model -> Movie -> Bool
+matchSelectedGenres model movie =
+    case Set.size model.selectedGenres of
+        0 ->
+            True
+
+        _ ->
+            Set.size (Set.intersect movie.genres model.selectedGenres) > 0
```

Comment:
If there are selectedGenres on the model, find the movies whose genres intersect with the selectedGenres list. Basically, makes selectedGenres an "any, or" on movies.